### PR TITLE
Enable live resizing for aui panels

### PR DESCRIPTION
**Pull request by Correct-Syntax**
Enables live resize for panels.

Unfortunately doing what I had originally planned with customizing the panels proved to be very difficult (requiring a ton of code to be changed and some things to be broken). This is mainly due to the fact that Berean uses AUI from wxwidgets rather than the pure-python version as I had hoped. Thus, there isn't the kind of customization ability I need with the current setup. (so I left off doing that).

Therefore this PR is the only (worthwhile) change I've made. I hope it helps. 🙂  

Blessings,
Noah

### CODE REVIEW

File: src/mainwindow.py [hunk 1]
```diff
@@ -50,7 +50,8 @@ def __init__(self, app):
         self.old_versions = []
         self.printing = html2.PrintingSystem(self)
 
-        self.aui = aui.AuiManager(self)
+        extra_flags = aui.AUI_MGR_LIVE_RESIZE
```
Comment (t1m0thyj) [inline src/mainwindow.py]: Should this be `aui.AUI_MGR_DEFAULT | aui.AUI_MGR_LIVE_RESIZE` to preserve any default flags?

**Review by t1m0thyj — COMMENTED** [review]
Thanks for taking a look at AUI enhancements, left a comment.

If you think it'd be better to use the pure-Python AUI version for customizability, I'm open to switching. 🙂

File: src/mainwindow.py (hunk 1 above)
Comment (Correct-Syntax) [reply src/mainwindow.py]: Maybe. It seems to behave the same either way but I added it in case.

**Comment by Correct-Syntax** [comment]
> If you think it'd be better to use the pure-Python AUI version for customizability, I'm open to switching. 

Yes, from my experience that would be the way to go (if a customized look is the goal). The difficulty is that in the case of Berean, its not just a drop-in replacement.

**Comment by t1m0thyj** [comment]
> Yes, from my experience that would be the way to go (if a customized look is the goal). The difficulty is that in the case of Berean, its not just a drop-in replacement.

Yeah, I've switched back and forth in the past: https://github.com/t1m0thyj/berean/commit/19b197aa0d5240e7b224f19d42ba9be6e12c33b2 In the end I think the reason I preferred the wxWidgets AUI was because it seemed more efficient being written in C++ and in recent versions it doesn't seem to be lacking any important feature of the Python AUI implementation anymore. But I'm still open to using either one as I said 🙂

**Pull request merged**